frontend/packages/dev-console/integration-tests/README.md

```diff
@@ -0,0 +1,38 @@
+# This file consists of guide lines to create automation scripts
+
+## Scenario files
+
+1. Only validations should be present
+2. Don't use test data directly - It needs to be passed via data files (will use .ts files for now )
+
+## View files
+
+1. Page objects should be id, css selectors, buttontext etc.. [No XPath]- Already following
+    * Each section should have one object as shown below (It helps to reduce the import list in scenarios)
+
+      export const deleteDeployPopupObj = {
+          form: element(by.css('form.modal-content')),
+          checkbox: element(by.css('input[type="checkbox"]')),
+          cancel: element(by.css('[data-test-id="modal-cancel-action"]')),
+          delete: element(by.css('#confirm-action'))
+      }
+
```

Comment:
You lost your backticks... they are needed to signify code blocks and it will keep formatting. Pretty sure you had them the last time I looked at it lol.

Reply:
Yeah, I removed them, because it is displaying lint errors

Reply:
Sorry? There are no lint errors in `.md` files... Our linter is `eslint`, the `es` is EcmaScript, which is the spec JavaScript is built on. Are you sure these were showing up as eslint errors?